src/components/Homepage.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import '../styles/Homepage.css';
+
+const HomePage = () => {
+  const categories = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    axios
+      .get('https://api.jikan.moe/v4/anime?q=&sfw')
+      .then((res) => {
+        const { data } = res.data;
+        dispatch({ type: 'SET_CATEGORIES', payload: data });
+      });
+  }, [dispatch]);
+
+  return (
+    <div className="grid-container">
+      {categories.map((category) => (
+        <Link to={`/details/${category.mal_id}`} key={category.mal_id} onClick={() => dispatch({ type: 'SET_SELECTED_CATEGORY', payload: category.mal_id })}>
+          <div className="category-item">
+            <img className="category-image" src={category.images.jpg.image_url} alt={category.image_url} />
+            <div className="category-title">{category.title}</div>
+          </div>
+        </Link>
+      ))}
+    </div>
```

Comment:
- [ ] Hi, you have done a nice job and your App is working fine. You need to work on the design and submit it for re-review. this is what the requirements say: 
- You should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)) including Layout: **composition and space between elements**.
NB: Pay attention to the blue marks in the pictures to understand what is missing in your design, and implement them if possible(Remove space in between pictures, also add those small arrows in each pictures)
- **Original**
![Capture1](https://github.com/Gardimy/React_Redux_capstone/assets/63972670/2482bac8-7e0e-41f4-8524-02614516ce33)
- Your design

![Capture](https://github.com/Gardimy/React_Redux_capstone/assets/63972670/1a6ce5d2-af9f-423d-a381-872c4dea80f1)